openshift/e2e-tests-openshift.sh

```diff
@@ -20,6 +21,19 @@ readonly OPENSHIFT_BUILD_NAMESPACE=${OPENSHIFT_BUILD_NAMESPACE:-tektoncd-build-$
 # test-git-volume: `"gitRepo": gitRepo volumes are not allowed to be used]'
 declare -ar SKIP_YAML_TEST=(test-git-volume)
 
+function replaceImagesWithCIBuilds() {
+  RELEASE_YAML=$1
+  REGISTRY_PREFIX=$2
+
+  # Substitute images using the images built by OpenShift CI
+  # Ref: https://github.com/openshift/release/pull/21940
+  sed -i -e "s|${REGISTRY_PREFIX}:tektoncd-triggers-controller|${TEKTONCD_TRIGGERS_CONTROLLER}|g" ${RELEASE_YAML}
```

Comment:
`TEKTONCD_TRIGGERS_CONTROLLER` to get ENV we need to have similar changes https://github.com/openshift/release/pull/21829/files for triggers also

Reply:
Ah i see its done as part of this PR https://github.com/openshift/release/pull/21940